.github/workflows/pull-request.yml

```diff
@@ -38,8 +38,12 @@ jobs:
       - name: Install dependencies
         run: pnpm install --frozen-lockfile
 
-      - name: Run prettier
-        run: pnpm run lint
+      - name: Setup Biome
+        uses: biomejs/setup-biome@v2
+        with:
+          version: latest
```

Comment:
Why do we need to setup Biome, when it is already a dependency in the project?

Reply:
I think we could get away without the setup-biome job but I like this approach better